web/docs/auth/auth-hooks.md

```diff
@@ -30,6 +32,17 @@ If you are using OAuth, the flow includes extra steps before the signup flow:
   caption="OAuth Flow with Hooks"
```

Comment:
I can't reach line 27, but we should update that one too probably (signup flow -> auth flow or something).